doc/src/site/sphinx/tutorials/aws_emr.rst

```diff
@@ -0,0 +1,64 @@
+Use Sparkling Water with Amazon EMR
+-----------------------------------
+
+To use Sparkling Water with an EMR cluster, you can use a premade H2O template.
+
+1. Log in to the `AWS Marketplace <https://aws.amazon.com/marketplace/>`__. 
+
+2. In the AMI & SaaS search bar, search for H2O, and select **H2O Artificial Intelligence** to open H2O in the marketplace. Review the information on this page. Note that the Delivery Methods section provides two options:
+
+   - **H2O Cluster of VM**: All the VPC, subnets, and network security groups are created for you.
+   - **H2O Cluster of VM - Bring your own VPC**: All the networks security groups, subnets, and internet gateways are created by the user.
+   
+   Click **Continue** after reviewing to open the Launch on EC2 page.
+
+  .. figure:: ../images/aws_h2oai.png
+      :alt: H2O Artificial Intelligence 
+
+3. The Launch on EC2 page provides information about launch options. On the Manual Launch tab:
+
+   - Select the offering that you prefer.
+   - Select the region to launch.
+   - Specify a Deployment option.
+
+   Click **Launch with CloudFormation Console** to begin creating your stack.
+
+  .. figure:: ../images/aws_launch_on_ec2.png
+     :alt: Launch On EC2 page
+
+4. On the Select Template page, enter https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/TemplateEMR.json in the **Specify an Amazon S3 template URL** field.
+
+ This template requires an install script, which is available here: https://s3-us-west-2.amazonaws.com/h2o-cloud/aws-template/install_h2o_sw.sh. This script can be added using one of the following methods:
+
+  - Download and place the script in the following s3 path: s3://h2o-cloud/aws-template/install_h2o_sw.sh
```

Comment:
@angela0xdata can you also please change `install_h2o_sw.sh` to `install_sparkling_water_SUBST_SW_VERSION.sh`

Reply:
already edited and the ^ as well, I was experimenting with the build

Thanks!